### PR TITLE
Update FAF Social rules link

### PR DESCRIPTION
**Pull request by EcoNooob**
idk how github works

### CODE REVIEW

**Review by Brutus5000 — APPROVED** [review]

File: templates/views/account/register.pug [hunk 1]
```diff
@@ -12,7 +12,7 @@ block content
           ul.text-left
             li <strong>You may only register one account.</strong> Additional accounts will be banned. If you misbehave using multiple accounts, you may be banned completely.
             li <strong>Your email address is the only way to regain access to your account.</strong> if you lose your password. If you lose both your password and your email, you will lose access to your account.
-            li <strong>Full FAF Social rules:</strong> <a href="https://forums.faforever.com/viewtopic.php?f=2&amp;t=581">FAF Lobby and Forum rules</a>
```
Comment (Brutus5000) [inline templates/views/account/register.pug:15]: I don' think the amp; is correct here.

**Review by Brutus5000 — CHANGES_REQUESTED** [review]
Misclicked.

File: templates/views/account/register.pug (hunk 1 above)
Comment (EcoNooob) [reply templates/views/account/register.pug:15]: Yeah I was wondering about that too, but the new link removes that part anyway?

Comment (louvenarde) [reply templates/views/account/register.pug:15]: change it, I was about to merge, now i can't

Comment (Brutus5000) [reply templates/views/account/register.pug:15]: Ha sorry. I shouldn't check that in a hurry. Yes you removed it. I thought you added it. But I tested it now without the amp; and it works as well.

**Review by Brutus5000 — APPROVED** [review]

**Pull request merged**